FE/todo-app/src/types/Auth.ts

```diff
@@ -0,0 +1,25 @@
+export type RegisterInputs = {
+  username: string;
+  email: string;
+  password: string;
+};
+
+export type LoginInputs = {
+  identifier: string;
+  password: string;
+};
+
+export type LoginResponse = {
+  jwt: string;
+  user: {
+    id: number;
+    username: string;
+    email: string;
+  };
+};
+
+export type LoginRequest = LoginInputs;
```

Comment:
tohle ne prosím, maximálně obráceně.. LoginInputs klidně ať jsou LoginRequest ale ne takhle.. když někdo přidá něco do loginInputů navíc co se potom nebude posílat na login ale někam jinam, tak ti to celé rozbije.